src/utils.rs

```diff
@@ -15,14 +39,18 @@ pub fn setup_hooks() {
     }
 }
 
-
 #[cfg(not(target_arch = "wasm32"))]
-pub fn get_arg(n: usize) -> Option<String> {
-    std::env::args().nth(n)
+pub fn get_arg(arg: MainArgs) -> Option<String> {
```

Comment:
converting this function to:
`pub fn get_args() -> CliArgs` (or `pub fn get_args() -> GaussianSplattingViewer` with the above suggestion) would simplify argument retrieval and avoid duplicate parse calls.